pkg/engine/engine.go

```diff
@@ -82,7 +78,8 @@ func (eng Engine) BuildImages(ctx context.Context, fn *function.Function) error
 		fmt.Println("\n" + cli.RenderError(err.Error()) + "\n")
 		os.Exit(1)
 	}
-	return ui.Builder.Error()
+
+	return ui.Error()
```

Comment:
Should we output to the logger that the functions were loaded/built successfully? The TUI seems to show "Exporting image..." as the last message, but some sort of final confirmation could be useful to tell the user it was successful